packages/devextreme/js/ui/drop_down_editor/ui.drop_down_editor.js

```diff
@@ -621,13 +621,14 @@ const DropDownEditor = TextBox.inherit({
         };
     },
 
-    _popupInitializedHandler: function() {
-        if(!this.option('onPopupInitialized')) {
-            return null;
-        }
+    _popupInitializedHandler: noop,
+
+    _getPopupInitializedHandler: function() {
+        const onPopupInitialized = this.option('onPopupInitialized');
 
         return (e) => {
-            this._popupInitializedAction({ popup: e.component });
+            this._popupInitializedHandler(e);
+            onPopupInitialized && this._popupInitializedAction({ popup: e.component });
```

Comment:
I believe it doesn't meet our code style. lets use if

Reply:
fixed